### PR TITLE
add pycbc live exes into pycbc

### PR DESCRIPTION
**Pull request by ahnitz**

### CODE REVIEW

**Comment by titodalcanton** [comment]
Alex, looks good, two suggestions:
* Can the new executables be added to the Travis `--help` test? Maybe we don't want to add MPI and cProfile to requirements.txt, but they can still be installed explicitly for Travis.
* Please add brief docstrings to the new scripts and pass them as descriptions to the ArgumentParser as usual.

**Comment by ahnitz** [comment]
@titodalcanton Thanks I'll add those suggestions soon. Still waiting on #1406 and #1407.

**Comment by ahnitz** [comment]
@titodalcanton We can add these packages into an optional set. I'm not sure people are well aware of this, but you have specificy optional package groups and if you use pip to install you can have it resolve those too. Something like

pip install pycbc[cuda]

should work to pull in pycbc with cuda support for example. We should do the same thing for PE  support, and MPI support, etc.

**Comment by ahnitz** [comment]
I don't think the Travis environment will have mpi, but I can check.

File: bin/pycbc_live [hunk 1]
```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python]
```
Comment (titodalcanton) [inline bin/pycbc_live]: Typo?

File: bin/pycbc_live [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+from time import time
+import os.path
+from mpi4py import MPI as mpi
+from pycbc.events import newsnr
+from pycbc.events.coinc import LiveCoincTimeslideBackgroundEstimator
+from pycbc.events.single import LiveSingleFarThreshold
+from pycbc.io.live import SingleCoincForGraceDB
+import pycbc.waveform.bank
+
+def makedir(path):
```
Comment (titodalcanton) [inline bin/pycbc_live]: I'd import this function from pycbc.workflow.core (also, the function will not happily ignore a None argument).

Comment (ahnitz) [inline bin/pycbc_live]: I've swapped the order so this applies the check in the right way. I'm not sure I want to pull in a Pegasus.DAX3 dependency here, but that does highlight a compartmentalization we should do in pycbc.workflow to separate out the things that don't require DAX3 support so we can import that alone.

**Comment by titodalcanton** [comment]
Thanks, I'll merge this once Travis is done.

**Pull request merged**